src/robot_dart/gui/magnum/base_application.cpp

```diff
@@ -176,7 +176,7 @@ namespace robot_dart {
 
                     /* Glyphs we need to render everything */
                     /* Latin characters for now only */
-                    _glyph_cache.reset(new Magnum::Text::DistanceFieldGlyphCache{Magnum::Vector2i{2048}, Magnum::Vector2i{512}, 22});
+                    _glyph_cache.reset(new Magnum::Text::DistanceFieldGlyphCache{Magnum::Vector2i{2048}, Magnum::Vector2i{512}, 32});
```

Comment:
Why did you change the size here?

Reply:
I wanted to have better glyphs, but it seems that it did not change anything.